src/PhoneInput.php

```diff
@@ -26,7 +26,10 @@ public function init()
         PhoneInputAsset::register($this->view);
         $id = ArrayHelper::getValue($this->options, 'id');
         $jsOptions = $this->jsOptions ? Json::encode($this->jsOptions) : "";
-        $this->view->registerJs("var input = $('#$id'); input.intlTelInput($jsOptions); input.parents('form').on('submit', function() { input.val(input.intlTelInput('getNumber')); });");
+        $this->view->registerJs("$('#$id').intlTelInput($jsOptions);");
+        if($this->hasModel()) {
+            $this->view->registerJs("$('#$id').parents('form').on('submit', function() { $('#$id').val(input.intlTelInput('getNumber')); })");
```

Comment:
It doesn't work for me so i replaced the last "input" => $('#$id') and then it works like charm